frontend_lib/test/setup.js

```diff
@@ -1,8 +1,11 @@
 import { JSDOM } from 'jsdom'
 import chai from 'chai'
+import sinon from 'sinon'
 import Enzyme from 'enzyme'
 import chaiEnzyme from 'chai-enzyme'
 import Adapter from 'enzyme-adapter-react-16'
+import EventSource from 'eventsourcemock'
+import AbortController from 'abort-controller'
```

Comment:
I think these dependencies should be added to frontend_lib/package.json

Reply:
See comment above.